functions.php

```diff
@@ -842,37 +842,87 @@ function display_rich_snippet( $content ) {
 		$args_video = get_option( 'bsf_video' );
 		$video      = '';
 
+		// Generate the schema structure.
```

Comment:
add this check here - 

if ( ! isset( $post->ID ) ) {
		return;
	}

Reply:
done @sushmak02